src/modules/pivotTable/__tests__/addToTotalIfNumber.js

```diff
@@ -0,0 +1,99 @@
+import { addToTotalIfNumber } from '../addToTotalIfNumber.js'
+
+const field = 'test'
+const tests = [
+    {
+        testName: 'negative value',
+        value: -1,
+        totalObj: {},
+        field,
+        expected: -1,
+    },
+    {
+        testName: 'zero value',
+        value: 0,
+        totalObj: {},
+        field,
+        expected: 0,
+    },
+    {
+        testName: 'positive value',
+        value: 1,
+        totalObj: {},
+        field,
+        expected: 1,
+    },
+    {
+        testName: 'null value',
+        value: null,
+        totalObj: {},
+        field,
+        expected: undefined,
+    },
+    {
+        testName: 'undefined value',
+        value: undefined,
+        totalObj: {},
+        field,
+        expected: undefined,
+    },
+    {
+        testName: 'string value',
+        value: 'string',
+        totalObj: {},
+        field,
+        expected: undefined,
+    },
+    {
+        testName: 'negative value with existing total',
+        value: -1,
+        totalObj: { [field]: 100 },
+        field,
+        expected: 99,
+    },
+    {
+        testName: 'zero value with existing total',
+        value: 0,
+        totalObj: { [field]: 100 },
+        field,
+        expected: 100,
+    },
+    {
+        testName: 'positive value with existing total',
+        value: 1,
+        totalObj: { [field]: 100 },
```

Comment:
is it necessary to use `totalObj` for the tests? Or could they just pass in e.g. `testName: 'foo', value: 1, total: 100, expected: 101`?